CONTRIBUTING.md

```diff
@@ -48,8 +48,8 @@ yarn test
 # Run the linter
 yarn lint
 
-# Run a command for the particular workspace (ex. @datadog/react-native)
-yarn workspace @datadog/react-native <command>
+# Run a command for the particular workspace (ex. @datadog/mobile-react-native)
+yarn workspace @datadog/mobile-react-native <command>
```

Comment:
now you have to do the change in the bridge generator also right ?

Reply:
no, it doesn't change the file path.